protocol/accessibility/README.md

```diff
@@ -0,0 +1,93 @@
+Accessibility
+=============
```

Comment:
I’m not sure we have a specific guideline around Markdown headings, but since we have level 3 headings in this document, thoughts on using ATX headings (the kind that uses `#` characters) throughout? Setext style headings are limited to only 2 levels, which is unfortunate.

Reply:
Yeah, I prefer ATX-style personally. I was trying to match what other files in this directory were using, but do you think we should switch to a new standard?